client/a8c-for-agencies/sections/marketplace/checkout/product-info.tsx

```diff
@@ -1,26 +1,56 @@
+import formatNumber from '@automattic/components/src/number-formatters/lib/format-number';
 import { useTranslate } from 'i18n-calypso';
+import pressableIcon from 'calypso/assets/images/pressable/pressable-icon.svg';
 import { useLicenseLightboxData } from 'calypso/jetpack-cloud/sections/partner-portal/license-lightbox/hooks/use-license-lightbox-data';
 import getProductIcon from 'calypso/my-sites/plans/jetpack-plans/product-store/utils/get-product-icon';
+import getPressablePlan from '../pressable-overview/lib/get-pressable-plan';
 import type { ShoppingCartItem } from '../types';
 
 export default function ProductInfo( { product }: { product: ShoppingCartItem } ) {
 	const translate = useTranslate();
 
 	const { title, product: productInfo } = useLicenseLightboxData( product );
 
-	if ( ! productInfo ) {
+	let productIcon =
+		productInfo?.productSlug && getProductIcon( { productSlug: productInfo?.productSlug } );
+	let productTitle = title;
+	let productDescription = productInfo?.lightboxDescription;
+
+	if ( product.family_slug === 'pressable-hosting' ) {
+		const presablePlan = getPressablePlan( product.slug );
+		if ( ! presablePlan ) {
+			return null;
+		}
+
+		productIcon = pressableIcon;
+		productTitle = product.name;
+		productDescription = translate(
+			'Plan with %(install)d WordPress installs, %(visits)s visits per month, and %(storage)dGB of storage per month.',
```

Comment:
Why do we override this information on the component level? Isn't it returned from the getProductIcon, productInfo, etc. similarly to how we visualize the other products?

Reply:
>  Isn't it returned from the getProductIcon, productInfo, etc., similar to how we visualize the other products?

No, it isn't returned from here as it only has products listed in the Marketplace. 

We currently have a difference in data structure for Pressable Plans.

While we could follow that approach by making all the necessary changes, there are a couple of reasons why I ended up choosing this approach over the one you mentioned:

1) `getProductInfo` gives a lot of other information, like `whatIsIncluded`, benefits, faqs, etc., which isn't necessary and is not used anywhere.
2) It is a bit complicated & time-consuming to achieve this.

I suggest we do this for now because of the time constraint and refactor this if needed. This means that if we decide to show more information about the product Currently, I see all the info for Pressable is hard coded at the component level. Let me know what you think.

Reply:
That makes sense! Thanks for the explanation.